src/Config.php

```diff
@@ -1567,89 +1567,58 @@ public static function displayPasswordSecurityChecks($field = 'password')
      *
      * @since 0.84
      *
-     * @param $password  string   password to validate
-     * @param $display   boolean  display errors messages? (true by default)
+     * @param string $password password to validate
+     * @param bool   $display  display errors messages? (true by default)
      *
      * @throws PasswordTooWeakException when $display is false and the password does not matches the requirements
      *
      * @return boolean is password valid?
+     *
+     * @deprecated 10.0.1
      **/
-    public static function validatePassword($password, $display = true)
-    {
+    public static function validatePassword(
```

Comment:
You should revert all changes made on this method. If it is deprecated, we should not start to change it and potentially introduce a bug.

Reply:
Done in 7b54c66ba8.